pimd/pim_vty.c

```diff
@@ -172,89 +172,66 @@ int pim_global_config_write_worker(struct pim_instance *pim, struct vty *vty)
 {
 	int writes = 0;
 	struct pim_ssm *ssm = pim->ssm_info;
-	char spaces[10];
 
-	if (pim->vrf->vrf_id == VRF_DEFAULT)
```

Comment:
is the indentation off now, for router pim blocks inside VRFs?

Reply:
VRF-aware PIM configuration is also at the top level.
```
router pim
  ...
router pim vrf blue
  ...
```